src/diffusers/pipelines/controlnet_xs/pipeline_controlnet_xs.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2024 The HuggingFace Team. All rights reserved.
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
I don't understand the code diff I'm seeing here 
this should be a new file, no?